lib/select-builder.js

```diff
@@ -62,28 +57,36 @@ const makeParamValue = (cond, value, params) => {
 class SelectBuilder extends QueryBuilder {
   constructor(params, options) {
     super(params, options);
-    this.operations = new Map();
-    for (const op of Object.keys(supportedOps)) {
-      const constructor = supportedOps[op];
-      this.operations.set(op, constructor ? new constructor() : null);
-    }
+    this.operations = {
+      select: new Set(),
+      selectDistinct: false,
+      innerJoin: [],
+      functions: [],
+      where: [],
+      groupBy: new Set(),
```

Comment:
We need array here, sequence matters

Reply:
@tshemsedinov, as I said in https://github.com/metarhia/sql/pull/12#discussion_r463952430, `Set`s are ordered according to the ECMAScript specification.

Reply:
Yeah, the array will look better here but the Set currently works the same since it also has the order (of insertion) and that's what we will use with Array anyway.